layout-webapp/src/main/resources/locale/portlet/SiteNavigation_fr.properties

```
@@ -1,6 +1,6 @@
 siteNavigation.button.tooltip.label=Navigation du site
 siteNavigation.drawer.title=Navigation du site
-siteNavigation.label.btn.createNode=Cr\u00e9er un noeud
+siteNavigation.label.btn.createNode=Ajouter un item
```

Should be managed by PO on crowdin even if it's possible to do with code

```suggestion
siteNavigation.label.btn.createNode=Cr\u00e9er un noeud
```

---

removed! 
cc @srenault-meeds FYI